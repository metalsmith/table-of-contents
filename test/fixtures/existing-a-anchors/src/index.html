---
layout: layout.hbs
toc: true
---
<a name="some-anchor-1"></a>
<h2>H2 heading</h2>
<a name="some-anchor-2"></a>
<h3>H3 heading</h3>
