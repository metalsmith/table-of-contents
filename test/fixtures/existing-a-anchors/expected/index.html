<ol class="toc">
  <li>
    <a href="#some-anchor-1">H2 heading</a>
    <ol class="toc">
      <li>
        <a href="#some-anchor-2">H3 heading</a>
      </li>
    </ol>
  </li>
</ol>

<a name="some-anchor-1"></a>
<h2>H2 heading</h2>
<a name="some-anchor-2"></a>
<h3>H3 heading</h3>

